root/static/scripts/tests/release-editor/actions.js

```diff
@@ -42,27 +42,55 @@ test('removing a medium should change the medium positions', function (t) {
   );
 
   var mediums = release.mediums();
-  t.deepEqual(mediums.map(x => x.position()), [1, 2, 3], 'medium positions are consecutive before removal');
+  t.deepEqual(
+    mediums.map(x => x.position()),
+    [1, 2, 3],
+    'medium positions are consecutive before removal',
+  );
 
   actions.removeMedium(mediums[0]);
-  t.deepEqual(mediums.map(x => x.position()), [1, 2], 'medium positions are consecutive after removal');
+  t.deepEqual(
+    mediums.map(x => x.position()),
+    [1, 2],
+    'medium positions are consecutive after removal',
+  );
 });
 
-test('reordering tracks that have non-consecutive "position" properties (MBS-7227)', function (t) {
-  t.plan(4);
-
-  var release = common.setupReleaseEdit();
-  var tracks = release.mediums()[0].tracks();
-  var originalTrack1 = tracks[0];
-  var originalTrack2 = tracks[1];
-
-  originalTrack2.position(3);
-
-  actions.moveTrackUp(originalTrack2);
-
-  t.equal(tracks[0], originalTrack2, 'original track 2 has moved to position 1');
-  t.equal(originalTrack2.position(), 1, 'original track 2 now has position() 1');
-
-  t.equal(tracks[1], originalTrack1, 'original track 1 has moved to position 2');
-  t.equal(originalTrack1.position(), 2, 'original track 1 now has position() 2');
-});
+test(
+  `reordering tracks that have non-consecutive
+   "position" properties (MBS-7227)`,
+  function (t) {
```

Comment:
This is unfortunate, since we had to rewrite all these files for the indent fixes PR. :( They were using four spaces before, and now we're changing them all back to that...

I think there is a way to avoid this by wrapping the first argument in parentheses, which I'd much prefer:

```JS
test((
  `reordering tracks that have non-consecutive
   "position" properties (MBS-7227)`
), function (t) {
  // this only requires two spaces now
});
```

Reply:
Made that change (which also thankfully made this PR a lot smaller)